test/e2e/kubelet_stats.go

```diff
@@ -157,8 +157,8 @@ func getContainerInfo(c *client.Client, nodeName string, req *kubelet.StatsReque
 		return nil, err
 	}
 	data, err := c.Post().
-		Prefix("proxy").
 		Resource("nodes").
+		SubResource("proxy").
```

Comment:
based on https://github.com/kubernetes/kubernetes/issues/18720, need to keep the old style if we want these tests to continue to run against a 1.0 server